README.md

```diff
@@ -98,8 +98,11 @@ stack build
 cd staging
 spago build
 
-# Ensure the compiled JavaScript is available to the client
-ln -s output ../client/public/js/output
+# Ensure the compiled JavaScript is available to the client via symbolic link.
+# Note: due to the differences in the `ln` command between
+# MacOS (BSD ln) and Linux (GNU ln), the below code will work
+# on both operating systems:
```

Comment:
We could probably omit this and just expect that folks copy/paste it.

Reply:
Could, but are we going to remember why this was needed? Do we need to remember?

Reply:
I do think this comment is a little bit odd - it's not due to the differences between the versions of `ln` that this command works on both mac and linux, it's _despite_ them! This command probably isn't going to change very often, and if it does break for one of the two OSes it's not the end of the world.

Reply:
Ok. I'll remove since both of you think it's not needed.